karmada/kcl.mod

```diff
@@ -0,0 +1,7 @@
+[package]
+name = "karmada"
+edition = "0.1.0"
+version = "0.1.0"
+
+[dependencies]
+k8s = "1.27"
```

Comment:
This model is generate with kcl-openapi, how can i get this k8s version? And i think i should be not push the files of `k8s/apimachinery/xxx`.

Reply:
> This model is generate with kcl-openapi, how can i get this k8s version? And i think i should be not push the files of `k8s/apimachinery/xxx`.

Yes, U can use the `kpm` tool to add this dependecy e.g., `kpm add k8s:1.27`. And U can remove the k8s folder.